x/leverage/simulation/operations_test.go

```diff
@@ -29,15 +29,32 @@ type SimTestSuite struct {
 // SetupTest creates a new umee base app
 func (s *SimTestSuite) SetupTest() {
 	checkTx := false
-	s.app = umeeappbeta.Setup(s.T(), checkTx, 1)
-	s.ctx = s.app.BaseApp.NewContext(checkTx, tmproto.Header{
-		ChainID: fmt.Sprintf("test-chain-%s", tmrand.Str(4)),
-		Height:  1,
-	})
+	betaApp := umeeappbeta.Setup(s.T(), checkTx, 1)
+	ctx := betaApp.NewContext(checkTx, tmproto.Header{})
+
+	// Note: Setting umee collateral weight to 1.0 to allow lender to borrow heavily
+	umeeToken := types.Token{
+		BaseDenom:            umeeapp.BondDenom,
+		ReserveFactor:        sdk.MustNewDecFromStr("0.25"),
+		CollateralWeight:     sdk.MustNewDecFromStr("1.0"),
+		BaseBorrowRate:       sdk.MustNewDecFromStr("0.02"),
+		KinkBorrowRate:       sdk.MustNewDecFromStr("0.2"),
+		MaxBorrowRate:        sdk.MustNewDecFromStr("1.0"),
+		KinkUtilizationRate:  sdk.MustNewDecFromStr("0.8"),
+		LiquidationIncentive: sdk.MustNewDecFromStr("0.1"),
+	}
+
+	betaApp.LeverageKeeper.SetRegisteredToken(ctx, umeeToken)
+	betaApp.LeverageKeeper.SetExchangeRate(ctx, umeeapp.BondDenom, sdk.MustNewDecFromStr("0.9"))
```

Comment:
```suggestion
```
This would be a `token:uToken` exchange rate, rather than an oracle price, so it should not need to be set. If it must be set though, make sure it is greater than 1.0

Reply:
done at https://github.com/umee-network/umee/pull/433/commits/9da105c9fdc1483edca89ca2b632fb333c21b56f